client/profile-wizard/style.scss

```diff
@@ -263,7 +263,7 @@
 			}
 
 			label.components-checkbox-control__label {
-				margin-left: $gap-large - $gap-smallest;
+				margin-left: $gap;
```

Comment:
This is just a small tweak to get the label to line up with the description per the design mockup